do/private/publish

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+pushd "${SCRIPT_DIR}/../.." > /dev/null 2>&1 || exit 1
+source "${SCRIPT_DIR}/script_utils"
+
+mkdir -p build
+
+check_env_var_with_pop INSTALLER_BUCKET_URL
+
+echo "--- :aws: Log into AWS Elastic Container Registry..."
+aws_login "us-east-2"
```

Comment:
Don't you want to log in to region 1? Or does it matter.

Reply:
No, because our `buildkite-spike` S3 bucket is in US-East-2.  Only using US-East-1 for Docker Images.